deck/packages/kubernetes/src/manifest/rollout/RollingRestart.tsx

```diff
@@ -15,32 +18,37 @@ interface IRollingRestartParameters {
   cloudProvider: string;
   location: string;
   manifestName: string;
+  reason?: string;
```

Comment:
This a new thing?  The "reaesn" on rolling restart operation? Seeing this in a few other places in this PR...